go/bundle/README.md

```diff
@@ -93,3 +93,52 @@ sign-bundle \
 ```
 dump-bundle -i foo.wbn
 ```
+
+## Using Bundles
+Bundles generated with `gen-bundle` can be opened with web browsers supporting web bundles.
+
+- Chrome (80+): enable `chrome://flags/#web-bundles` flag to enable experimental support for Web Bundles.
```

Comment:
All these problems below are only for Chrome.
I think we should have a doc about these information in chromium repository, and this document should have a link to that.
So can we can update the information when we change the implementation synchronously.

How about this?

Chrome (80+) experimentally supports Web Bundles with some limitations. See [this document](https://chromium.googlesource.com/chromium/src/+/refs/heads/master/content/browser/web_package/web_bundles.md) for more details.

Reply:
Having this doc in chromium repository and linking from here sounds good, so +1

Reply:
OK, created https://chromium-review.googlesource.com/c/chromium/src/+/1880413.

Reply:
Done